pkgs/development/ocaml-modules/x509/default.nix

```diff
@@ -8,11 +8,11 @@ buildDunePackage rec {
   minimumOCamlVersion = "4.07";
 
   pname = "x509";
-  version = "0.11.0";
+  version = "0.11.1";
 
   src = fetchurl {
```

Comment:
Please use `fetchFromGitHub` or `fetcharchive` instead of `fetchurl` on a Github archive; see https://github.com/NixOS/nix/issues/1735#issuecomment-352082326

Reply:
Doesn't apply here, since the downloaded file is a release asset which is uploaded by the maintainer and not generated by GitHub. Therefore its hash is stable. Release assets can't be fetched using `fetchFromGitHub`. See #85289.